lib/liquid_voting_web/resolvers/voting.ex

```diff
@@ -28,11 +28,10 @@ defmodule LiquidVotingWeb.Resolvers.Voting do
     do: {:ok, Voting.list_votes_by_proposal(proposal_url, organization_id)}
 
   def votes(_, _, %{context: %{organization_id: organization_id}}) do
-    Tracer.with_span "resolvers/voting" do
+    Tracer.with_span "#{__MODULE__} #{inspect(__ENV__.function)}" do
       Tracer.set_attributes([
-        {:action, "votes"},
         {:request_id, Logger.metadata()[:request_id]},
-        {:organization_id, organization_id}
+        {:vars, binding()}
```

Comment:
which vars are being sent, here?

Reply:
All the args passed into the function and their values.  

For example for `create_vote`, we get:
```
[args: %{participant_email: "rrr@somedomain.com", proposal_url: "https://other_one", yes: true}, email: "rrr@somedomain.com", organization_id: "b7a9cae5-6e3a-48b1-8730-8b5c8d6c9b5a"]
```

Reply:
I was going with the principle that as much info. as possible is preferable.

Reply:
Ok, I was worried we'd be getting more internal stuff. 

But vars[:args][:participant_email] is the same as vars [:email], and we also need to access this atomically in order to anonymise it. I know it's verbose, but I think it's better to access each one of these atomically. 

This will also make it more explicit on what we're trying to track